R/theme-preview.R

```diff
@@ -43,12 +46,16 @@ colorpicker_deps <- function() {
   )
 }
 
-opts_metadata <- jsonlite::fromJSON(system.file("themer/options.json", package = "bootstraplib"),
```

Comment:
Using `system_file` saved us from a bad build-time bug that would have bitten us after it went to CRAN. 😬 

When I was building and installing the package after adding `system_file`, I started getting this error:

```
** testing if installed package keeps a record of temporary installation path
ERROR: hard-coded installation path: please report to the package maintainer and use ‘--no-staged-install’
```

It's because the package is calling `system_file()` at build time, and the local path is getting stored in `system_file`'s cache.

The problem actually did occur before I added `system_file`, but because the local path got stored as part of a JSON object, R didn't detect that the path got embedded in there. With the path cached in `system_file`, it was able to detect the problem. Whew!

Reply:
Good catch!!